README.md

```diff
@@ -1,13 +1,6 @@
-opx-db-sql
-==========
-opx-db-sql is wrapper to db-sql api used for VM target.
+# opx-db-sql
+This repository contains a wrapper to the db-sql API which is used for a virtual machine (VM) target.
 
-Description
------------
-VM presents an abstraction of the hardware (thermal sensors, fans,
-LEDs, optics,interfaces, etc.) in a VM environment. SDI-VM's sole
-client is PAS, similarly SAI-VM client is NAS and it queries the
-devices using the API defined in sdi-api and functionality in sai-api.
-
-(c) Dell 2016
+A VM presents an abstraction of the hardware (thermal sensors, fans, LEDs, optics, interfaces, and so on) in a VM environment. SDI VM's sole client is PAS, similarly the SAI VM client is NAS and it queries the devices using the API defined in the sdi-api and functionality in the sai-api.
```

Comment:
I missed this: the last sentence of this paragraph should not be there  ... SDI VM's sole client is PAS ...